Description: Upstream changes introduced in version 3.0.7-1atrust2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 gst-fsl-plugins (3.0.7-1atrust2) t66; urgency=low
 .
   * Add missing runtime dependencies
 .
 The person named in the Author field signed this changelog entry.
Author: Atrust Corp (Atrust Development Team) <atrust@atrustcorp.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ gst-fsl-plugins-3.0.7/autoregen.sh
@@ -0,0 +1,2 @@
+#!/bin/sh
+./autogen.sh PLATFORM=MX6 $@
